File: PHP-COPY-JS/11-grading-conversion.php
<?php

function gradingStudents($grades)
{
    // Remove the first element
    array_shift($grades);

    // Loop through the remaining grades
    for ($i = 0; $i < count($grades); $i++) {
        $grade = $grades[$i];
        if ($grade >= 38) {
            $diferencaPorCinco = 5 - ($grade % 5);
            if ($diferencaPorCinco < 3) {
                $grades[$i] = $grade + $diferencaPorCinco;
            }
        }
    }

    print_r($grades);
    return $grades;
}

// Example usage
gradingStudents([4, 73, 67, 38, 33]);

File: PYTHON3-COPY-JS/11-grading-conversion.py
def grading_students(grades):
    # Remove the first element
    grades = grades[1:]

    # Loop through the remaining grades
    for i in range(len(grades)):
        grade = grades[i]
        if grade >= 38:
            diferenca_por_cinco = 5 - (grade % 5)
            if diferenca_por_cinco < 3:
                grades[i] = grade + diferenca_por_cinco

    print(grades)
    return grades

# Example usage
grading_students([4, 73, 67, 38, 33])
